tiatoolbox/utils/misc.py

```diff
@@ -235,3 +237,35 @@ def mpp2objective_power(mpp):
         # Double the objective power as mpp halves
         return 10 * 2 ** (np.ceil(np.log2(0.15 / mpp)) + 2)
     raise ValueError()
+
+
+def contrast_enhancer(img, low_p=2, high_p=98):
+    """Enhancing contrast of the input RGB image using intensity adjustment.
+       This method uses both image low and high percentiles.
+
+    Args:
+        img (ndarray): input image used to obtain tissue mask.
+            Image should be RGB uint8.
+        low_p (scalar): low percentile of image values to be saturated to 0.
+        high_p (scalar): high percentile of image values to be saturated to 255.
+            high_p should always be greater than low_p.
+
+    Returns:
+        img (ndarray): Image uint8 RGB with contrast enhanced.
+
+    Examples:
+        >>> from tiatoolbox import utils
+        >>> img = utils.misc.contrast_enhancer(img, low_p=2, high_p=98)
+
+    """
+    # check if image is not uint8
+    if not img.dtype == np.uint8:
+        raise Exception("Image should be RGB uint8.")
+    img_out = img.copy()
+    p_low, p_high = np.percentile(img_out, (low_p, high_p))
+    if p_low >= p_high:
+        p_low, p_high = np.min(img_out), np.max(img_out)
```

Comment:
@mostafajahanifar can you check if we can improve code coverage by adding a test?

Reply:
done!